tests/test_pipeline_mixin.py

```diff
@@ -937,9 +937,13 @@ def compare_pipeline_args_to_hub_spec(pipeline_class, hub_spec):
     hub_args = set(get_arg_names_from_hub_spec(hub_spec))
 
     # Special casing: We allow the name of this arg to differ
-    js_generate_args = [js_arg for js_arg in hub_args if js_arg.startswith("generate")]
+    js_generate_args = [
+        js_arg for js_arg in hub_args if js_arg.startswith("generate") or js_arg.startswith("generation")
```

Comment:
just wondering what this `js` means .

Reply:
and it would be nice to have a comment of what 

> compare_pipeline_args_to_hub_spec

is trying to do 🙏

Reply:
I think renaming `js_generate_args` to `hub_generate_args` would be more consistent (can be done here or separate PR). I believe the `js_` comes from the fact that these specs live in the `huggingface.js` repo as JSONschema specs.

Reply:
Will update both!